src/theme.scss

```diff
@@ -3342,7 +3342,7 @@ body.is-tablet {
 .backlink-pane,
 .outgoing-link-pane {
   padding-bottom: unset;
-  overflow: visible;
+  overflow: scroll;
```

Comment:
`scroll` adds a margin to the right and to the bottom, to reserve place for both horizontal and vertical scrollbars

it's a bit unnecessary, so i would suggest to put it to `auto` instead (default value for most cases)